Detectors/TRD/workflow/src/TRDTrapSimulatorSpec.cxx

```diff
@@ -560,6 +585,7 @@ o2::framework::DataProcessorSpec getTRDTrapSimulatorSpec()
                              {"trd-printtrapconfig", VariantType::Bool, false, {"Name of the trap config from the CCDB"}},
                              {"trd-drawtracklets", VariantType::Int, 0, {"Bitpattern of input to TrapSimulator Draw method (be very careful) one file per track"}},
                              {"trd-printtracklets", VariantType::Int, 0, {"Bitpattern of input to TrapSimulator print method"}},
+                             {"trd-fixtrigrecord", VariantType::Bool, false, {"Fix trigger record alignment, temporary, hence false by default"}},
```

Comment:
Should this not also be `trd-fixtriggerrecord`?

Reply:
i can change it, i just wanted to type less ;-)

Reply:
ah fixed.